wp/lib/bap_wp/src/precondition.mli

```diff
@@ -51,7 +51,7 @@ val z3_expr_zero : Z3.context -> int -> Constr.z3_expr
 val z3_expr_one : Z3.context -> int -> Constr.z3_expr
 
 (** Translate a BIR binary operator to a Z3 one. *)
-val binop : Z3.context -> Bap.Std.binop -> Constr.z3_expr -> Constr.z3_expr -> Constr.z3_expr
+val binop : Env.t -> Bap.Std.binop -> Constr.z3_expr -> Constr.z3_expr -> Constr.z3_expr
 
 (** Translate a BIR unary operator to a Z3 one. *)
 val unop : Z3.context -> Bap.Std.unop -> Constr.z3_expr -> Constr.z3_expr
```

Comment:
Not super important, but what are your thoughts on changing the argument from `context` to `env` in `unop` so that it continues to mirror the api for `binop`?

Reply:
Hmm. I remember this being annoying and unsatisfying. I think maybe passing in context and an optional smt-compat bool flag might be better. Passing in the env is overkill

Reply:
Sounds good! We can leave this as is.